style.css

```diff
@@ -458,3 +470,191 @@
 .icon {
   margin-right: 20px;
 }
+
+.contact {
+  width: 100%;
+  background: #6070ff;
+  background-image: url(.Contact-me.png);
+  background-repeat: no-repeat;
+  background-size: contain;
+  background-position: 143px 11px;
+  border-top-left-radius: 80px;
+  margin-top: -70px;
+  padding: 103px 12px;
```

Comment:
- [ ] Like the previous reviewer mentioned please make sure your contact form looks as close to the Figma design as possible 🙏 

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/177765225-8159fe32-7628-4260-be5d-b132a5c8d1a7.png)  |  ![127 0 0 1_5501_ (1)](https://user-images.githubusercontent.com/74814780/177765264-bc7f4635-88fa-4dce-8066-6b013bb7146c.png)